scripts/zones/Inner_Horutoto_Ruins/IDs.lua

```diff
@@ -45,6 +45,8 @@ zones[xi.zone.INNER_HORUTOTO_RUINS] =
             [17563799] = 17563801, -- -237.426 0.5 -23.412
             [17563800] = 17563801, -- -230.732 -0.025 -52.324
         },
+        MAGICKED_BONES_PH_CLUB = GetFirstID("Magicked_Bones") - 2,
```

Comment:
why do you have -2 here and then adding +2 over in the mob script?

And then minus one on the other. You don't even need both entries. The function will return the 1st magic bones ID it finds, with no +/- needed. You then "+1" to get the _second_ bones ID.

Reply:
I posted the code for that last night.